lib/wallaby/experimental/chrome/chromedriver/server.ex

```diff
@@ -54,27 +54,25 @@ defmodule Wallaby.Experimental.Chrome.Chromedriver.Server do
 
   @impl true
   def init({chromedriver_path, opts}) do
-    port_number = Utils.find_available_port()
-
     startup_timeout = Keyword.get(opts, :startup_timeout, @default_startup_timeout)
     Process.send_after(self(), :ensure_readiness, startup_timeout)
 
-    send(self(), :start_chromedriver)
-
-    {:ok, %State{port_number: port_number, chromedriver_path: chromedriver_path}}
+    {:ok, %State{chromedriver_path: chromedriver_path}, {:continue, nil}}
   end
 
   @impl true
-  def handle_info(:start_chromedriver, state) do
-    %State{port_number: port_number, chromedriver_path: chromedriver_path} = state
+  def handle_continue(_, state) do
```

Comment:
I think I would still pass the atom `:start_chromedriver` to the continue callback, just to be explicit.

Reply:
Thanks for pointing this out. I thought this callback only ran after `init/1`, but it looks like you can return a continue from `handle_call/3` and `handle_cast/2` as well. Updating...